vars/daosPackagesVersion.groovy

```diff
@@ -0,0 +1,96 @@
+// vars/daosPackagesVersion.groovy
+
+/**
+ * daosPackagesVersion.groovy
+ *
+ * daosPackagesVersion variable
+ */
+
+/**
+ * Method to return the version of the DAOS packages
+ */
+
+import groovy.transform.Field
+
+@Field static rpm_version_cache = ""
+
+String daos_latest_version(String next_version) {
+    String v = sh(label: "Get RPM packages version",
+                  script: 'repoquery --repofrompath=daos,' + env.REPOSITORY_URL +
+                           env.DAOS-STACK-EL-7-STABLE-REPO + '''/ \
+                                     --repoid daos --qf %{version}-%{release} --whatprovides 'daos(x86-64) < ''' +
+                                     next_version + '''' |
+                                 rpmdev-sort | tail -1''',
+                  returnStdout: true).trim()
```

Comment:
Does this fix the issue that I am having with https://github.com/daos-stack/daos/pull/5116 that forced me to use yum by removing the trailing newline?

Reply:
I don't know.  I have not seen what the issue looks like or how it manifests itself.  I really need to see a run with it exhibiting the bad behaviour to know.